### ISSUE TITLE
Warning when using wrong comment syntax

### ISSUE THREAD

**Issue by PatrickRose** [koalaman/shellcheck#1076]
#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/sh
// I am an idiot who forgets comment syntax in different languages
# This is actually a comment
```

#### Here's what shellcheck currently says:
No errors

#### Here's what I wanted or expected to see:

Something to tell me off for trying to do that - it's obviously wrong because it'll expand out to the root directory and try and execute it. Which isn't possible. Can shellcheck warn about this?

**Comment by arth1** [comment]
Dupe of #865 "Warn on non-shell comments", I think.

**Comment by koalaman** [comment]
This has been fixed for #865, thanks!
